traveler-ios-ui/TravelerKitUI/ViewControllers/LoadingViewController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  LoadingViewController.swift
+//  TravelerKitUI
+//
+//  Created by Rakin Hoque on 2019-11-08.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+
+open class LoadingViewController: UIViewController {
+    @IBOutlet weak var loadingTitle: UILabel!
+    
+    var loadingTitleString: String?
+    
+    public override func viewDidLoad() {
+        if let loadingTitleString = loadingTitleString {
```

Comment:
Don't need this `if let`.

Reply:
Don't want to override default storyboard text unless loadingTitleString is set